layer/src/test/scala/geotrellis/raster/geotiff/GeoTiffReprojectRasterSourceSpec.scala

```diff
@@ -59,22 +59,24 @@ class GeoTiffReprojectRasterSourceSpec extends FunSpec with RasterMatchers with
       }
 
       val twiceFuzzySource = rasterSource.reprojectToGrid(LatLng, twiceFuzzyLayout).asInstanceOf[GeoTiffReprojectRasterSource]
-      twiceFuzzySource.closestTiffOverview.cellSize shouldBe CellSize(20,20)
+      twiceFuzzySource.closestTiffOverview.cellSize shouldBe CellSize(20, 20)
 
       val thriceFuzzyLayout = {
         val CellSize(width, height) = baseReproject.cellSize
         LayoutDefinition(RasterExtent(LatLng.worldExtent, CellSize(width*3.5, height*3.5)), tileSize = 256)
       }
 
       val thriceFuzzySource = rasterSource.reprojectToGrid(LatLng, thriceFuzzyLayout).asInstanceOf[GeoTiffReprojectRasterSource]
-      thriceFuzzySource.closestTiffOverview.cellSize shouldBe CellSize(20,20)
+      thriceFuzzySource.closestTiffOverview.cellSize.width shouldBe 40.0 +- 0.1
+      thriceFuzzySource.closestTiffOverview.cellSize.height shouldBe 40.0 +- 0.1
```

Comment:
This change is necessary as the former expectation (always round up to higher resolution levels) was baked in and doesn't correctly track GDAL behavior

Reply:
These are AutoHigher tests, let's keep them working with AutoHigher strategy? And this one cxan be a new test (this is just a sainity check that we could preserve the old behavior), how do you feel about it?